src/kit/LogViewer/LogViewerEntry.tsx

```diff
@@ -12,6 +12,7 @@ export interface LogEntry {
   formattedTime: string;
   level?: LogLevel;
   message: string;
+  htmlMessage?: boolean;
```

Comment:
q: why are we exposing the ability to directly insert HTML into the component? as the name suggests, `dangerouslySetInnerHTML` [is dangerous](https://react.dev/reference/react-dom/components/common#dangerously-setting-the-inner-html), if anything we should add more sanitization steps between the message and being displayed, not remove them.

Reply:
I was trying to add the highlight color to specific section of log text. I was basically adding classname to a section of message.